Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM calico/build:latest
+MAINTAINER projectcalico
+
+ADD . /code
+WORKDIR /code
+
+ARG K8S_VERSION=local
+ENV K8S_VERSION ${K8S_VERSION}
+
+RUN pip install pykube && pyinstaller calico.py -ayF && \
```

Comment:
I suppose this does mean we can't do an automated DockerHub build, but I'm worried that the binaries built here could be different from the binaries actually included in a release.